examples/src/main/java/org/apache/flink/FlinkDemo.java

```diff
@@ -0,0 +1,261 @@
+import org.apache.flink.connector.nebula.connection.NebulaConnectionProvider;
+import org.apache.flink.connector.nebula.connection.NebulaGraphConnectionProvider;
+import org.apache.flink.connector.nebula.connection.NebulaMetaConnectionProvider;
+import org.apache.flink.connector.nebula.sink.AbstractNebulaOutPutFormat;
+import org.apache.flink.connector.nebula.sink.NebulaBatchOutputFormat;
+import org.apache.flink.connector.nebula.sink.NebulaSinkFunction;
+import org.apache.flink.connector.nebula.source.NebulaInputFormat;
+import org.apache.flink.connector.nebula.source.NebulaSourceFunction;
+import org.apache.flink.connector.nebula.statement.ExecutionOptions;
+import org.apache.flink.runtime.state.filesystem.FsStateBackend;
+import org.apache.flink.streaming.api.CheckpointingMode;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.datastream.DataStreamSource;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.streaming.api.functions.source.SourceFunction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class FlinkDemo {
+    private static final Logger LOG = LoggerFactory.getLogger(FlinkDemo.class);
+
+    private static final String ADDRESS = "127.0.0.1:45500";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "nebula";
+    private static final String NAMESPACE = "nb";
+    private static final String LABEL = "player";
+    private static final ExecutionOptions sourceExecutionOptions;
+    private static final ExecutionOptions sinkExecutionOptions;
+    private static final NebulaConnectionProvider graphConnectionProvider;
+    private static final NebulaConnectionProvider metaConnectionProvider;
+
+    static {
+        NebulaClientOptions nebulaClientOptions = new NebulaClientOptions
+                .NebulaClientOptionsBuilder()
+                .setAddress("127.0.0.1:3699")
+                .build();
+        graphConnectionProvider = new NebulaGraphConnectionProvider(nebulaClientOptions);
+
+        NebulaClientOptions nebulaClientOptions1 = new NebulaClientOptions
+                .NebulaClientOptionsBuilder()
+                .setAddress("127.0.0.1:45500")
+                .build();
+        metaConnectionProvider = new NebulaMetaConnectionProvider(nebulaClientOptions1);
+
+        List<String> cols = new ArrayList<>();
+        cols.add("name");
+        cols.add("age");
+        sourceExecutionOptions = new ExecutionOptions.ExecutionOptionBuilder().setDataType("VERTEX")
+                .setGraphSpace("flinkSource")
+                .setLabel(LABEL)
+                .setFields(cols)
+                .setLimit(100)
+                .builder();
+        sinkExecutionOptions = new ExecutionOptions.ExecutionOptionBuilder().setDataType("VERTEX")
+                .setGraphSpace("flinkSink")
+                .setLabel(LABEL)
```

Comment:
schema maybe better than label?

Reply:
it means tag or edge, why schema?

Reply:
split ExcutionOptions into EdgeExecutionOptions and VertexExecutionOptions